### PR TITLE
Update Jenkinsfile

### PR DESCRIPTION
**Pull request by sebastienvas**

### CODE REVIEW

**Comment by sebastienvas** [comment]
build istio/manager-pr-master

File: Jenkinsfile [hunk 1]
```diff
@@ -78,7 +78,7 @@ def stablePresubmit(gitUtils, bazel, utils) {
     sh('ln -s ~/.kube/config platform/kube/')
     stage('Integration Tests') {
       timeout(30) {
-        sh("bin/e2e.sh -count 10 -debug -tag debug${gitUtils.GIT_SHA} -v 2")
```
Comment (kyessenov) [inline Jenkinsfile]: these are different images, don't conflate them under one SHA.

Comment (sebastienvas) [inline Jenkinsfile]: it won t be the same SHA as one SHA is from a PR to master, while the second will be the SHA after merge

Comment (kyessenov) [inline Jenkinsfile]: Which ones are being pushed to docker.io though?

Comment (sebastienvas) [inline Jenkinsfile]: it is not great, but otherwise I need to pass in another build parameter to istio/istio Jenfkinsfile.

Comment (sebastienvas) [inline Jenkinsfile]: we are rebuilding them at https://github.com/istio/manager/blob/master/Jenkinsfile#L95

Comment (kyessenov) [inline Jenkinsfile]: Ok, that's ok then. I still wonder what's the difference between post-submit and staging from master...

**Pull request merged**